### ISSUE TITLE
Merge this repo with `brainglobe-meta`

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/BrainGlobe#81]
The purposes of the two repos could be easily merged, and we'd have one less repo to think about.

**Comment by alessandrofelder** [comment]
Agreed, maybe we want two different project boards (or labels?) in the merged repo
- one for high-level abstract issues pertaining to the whole of BrainGlobe (the purpose of issues here)
- one for meta-package related, concrete issues (the usual repo-level issue-tracking)

I think this distinction was the reason we went for separate repos originally.

**Comment by adamltyson** [comment]
We could just add tags? To me the metapackage is the natural place to have these high-level issues anyway.

**Comment by adamltyson** [comment]
Might not be a good idea, but I'm always looking for ways to simplify the developer experience.

**Comment by alessandrofelder** [comment]
I don't have a strong opinion either way. Only that we document this (which I don't thing we do currently 😅 )

**Comment by IgorTatarnikov** [comment]
@adamltyson, should we go ahead with merging the two repos or close as not planned?

**Comment by adamltyson** [comment]
Not sure, i guess we need a bit more input from @brainglobe/active-devs

**Comment by IgorTatarnikov** [comment]
Based on conversations with @alessandrofelder we think it's easier to leave as is. The `brainglobe-meta` package should soon have issues related to its ongoing development which may end up polluting this repo, or at the minimum add extra overhead by needing to tag all new issues. Maintaining both repos hasn't added a lot of extra work, so it may not be worth going through the effort of joining.

**Comment by adamltyson** [comment]
Sounds good
